packages/integrations/netlify/package.json

```diff
@@ -23,6 +23,8 @@
     "./netlify-functions.js": "./dist/netlify-functions.js",
     "./edge-functions": "./dist/integration-edge-functions.js",
     "./netlify-edge-functions.js": "./dist/netlify-edge-functions.js",
+    "./polyfill.js": "./dist/polyfill.js",
+    "./polyfill": "./dist/polyfill.js",
```

Comment:
Are these two lines needed in this change? I can't seem to find the new file in dist

Reply:
Oops, left over from a previous attempt at making this work. Good catch, will update